# Data that client send
int32 num1
int32 num2
int32 num3
---
# Server response
int32 sum